src/plugins/intel_gpu/tests/test_cases/reorder_gpu_test.cpp

```diff
@@ -2818,6 +2814,8 @@ TEST(reorder_onednn_gpu, basic_convert_int8) {
 
 #ifdef RUN_ALL_MODEL_CACHING_TESTS
 TEST(reorder_gpu_optimization, compare_with_ref__b_fs_yx_fsv32_to_bfyx_f32_cached) {
+    if (get_test_engine().get_device_info().supports_immad)
+        return;
```

Comment:
let's separate such test removal from this PR so that we can track changes like this later.